src/main/java/com/exasol/projectkeeper/validators/dependencies/DependenciesValidator.java

```diff
@@ -39,7 +42,9 @@ public DependenciesValidator(final MavenFileModelReader fileModelReader,
     @Override
     public Validator validate(final Consumer<ValidationFinding> findingConsumer) {
         final List<ProjectDependency> dependencies = this.projectDependencyReader.readDependencies(this.pomFile);
-        final String expectedDependenciesPage = new DependencyPageRenderer().render(dependencies);
+        final List<ProjectDependency> dependenciesWithFixedLinks = new DependenciesBrokenLinkReplacer(
```

Comment:
It's strange somehow that you're fixing dependencies in the validate() method. I think this should happen somewhere during building the model after you read the dependencies. Any chances to move this logic?